src/forwardlabelingoptimizer.jl

```diff
@@ -147,33 +219,46 @@ function run_forward_labeling(net, dbg_sol = Vector{Int}())
             edge_cost = net.edge_costs[edge_pos]
             dst_consumption_lb = net.vertex_consumptions_lb[dst.v]
             dst_consumption_ub = net.vertex_consumptions_ub[dst.v]
-            for label in src.active_labels
+
+            # iterate over the labels of node $src
```

Comment:
adding precision since the code evolved
```suggestion
            # iterate over the labels of node $src to be extended
```